scdrs/util.py

```diff
@@ -568,10 +568,10 @@ def plot_group_stats(
             and (df_hetero_fdr is not None)
         ), "If dict_df_stats is not provided, df_fdr_prop, df_assoc_fdr, df_hetero_fdr must be all provided."
 
```

Comment:
add two lines when `df_assoc_fdr` and `df_hetero_fdr` are directedly passed in:
1. `df_assoc_fdr = (df_assoc_fdr < assoc_fdr_threshold)`
2. `df_hetero_fdr = (df_hetero_fdr < hetero_fdr_threshold)`

Reply:
Thank you, I missed it.

Reply:
Worked with the code below.

```
df_fdr_prop = pd.DataFrame([[0.1, 0.2, 0.3], [0.1, 0.2, 0.001]], 
                             index=['A', 'B'], columns=['0', '1', '2'])
df_assoc_fdr = pd.DataFrame([[0.1, 0.2, 0.03], [0.1, 0.2, 0.003]], 
                            index=['A', 'B'], columns=['0', '1', '2'])
df_hetero_fdr = pd.DataFrame([[0.1, 0.002, 0.03], [0.0001, 0.2, 0.003]], 
                             index=['A', 'B'], columns=['0', '1', '2'])

scdrs.util.plot_group_stats(df_fdr_prop=df_fdr_prop, 
                            df_assoc_fdr=df_assoc_fdr, df_hetero_fdr=df_hetero_fdr)
```